### ISSUE TITLE
[A11y] Mobile Collection Landing Image thumbnails

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#2006]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

No

### Issue Description

Collection Landing: https://staging.oregondigital.org/collections/gwilliams

In mobile view the image thumbnails are extra small, it will even be smaller in 320 view. See screen snap
![collection1](https://user-images.githubusercontent.com/8353860/130878388-9540eab6-afb4-4eba-a568-a4bde7542a8a.png)

In the desktop version, on hover the images have a black outline with title. However in mobile view you don't see that on hover only on click and it is too small to even read. See screen snap.
![collection2](https://user-images.githubusercontent.com/8353860/130878475-40a2e33b-bf7e-452e-a7b9-d8b245a9e0d8.png)

### Can anybody QA or does this need an expert?

Anybody
